pkg/scaffold/api.go

```diff
@@ -153,10 +152,10 @@ func (api *API) scaffoldV1() error {
 			fmt.Sprintf("%s_controller_test.go", strings.ToLower(r.Kind))))
 
 		err := (&Scaffold{}).Execute(api.buildUniverse(), input.Options{},
-			&controller.Controller{Resource: r},
-			&controller.AddController{Resource: r},
-			&controller.Test{Resource: r},
-			&controller.SuiteTest{Resource: r},
+			&controllerv1.Controller{Resource: r},
```

Comment:
it could be ctrlv1 and ctrlv2 instead of.

Reply:
It could, but IMHO it is not used that many times to justify loosing the clarity the full name gives to type 6 letters less.